stable/pipelines/templates/mysql/configmap-kf.yaml

```diff
@@ -31,28 +31,23 @@ data:
       touch $LOCK_FILE
     fi
 
+
   v3io-mysql.sh: |
     #!/usr/bin/env bash
     set -e
 
     echo "Current user -> $(whoami)"
 
-    LOCK_FILE=/var/lib/mysql/.allow-remote-access-complete
-    if [ ! -f $LOCK_FILE ]
-    then
-      MYSQL_CONFIG=/etc/mysql/my.cnf
-      INIT_SCRIPT="/etc/config/mysql/init-scripts/enable-root-remote-access.sql"
+    MYSQL_SOCKET_FILE=/var/run/mysqld/mysql.sock
+    INIT_SCRIPT="/etc/config/mysql/init-scripts/enable-root-remote-access.sql"
 
-      # Call init script on startup
-      echo "Adding $INIT_SCRIPT to $MYSQL_CONFIG ..."
-      echo "" >> $MYSQL_CONFIG
-      echo "[mysqld]" >> $MYSQL_CONFIG
-      echo "init-file=$INIT_SCRIPT" >> $MYSQL_CONFIG
+    echo "Starting MySQL ..."
 
-      touch $LOCK_FILE
-    fi
+    # TODO: review the innodb flags, they might not be needed anymore
+    mysqld \
+      --user={{- include "pipelines.db.DBRunUser" . }} \
+      --sql_mode="" \
+      --init-file=$INIT_SCRIPT \
+      --socket=$MYSQL_SOCKET_FILE
```

Comment:
Why did you add sql mode and the socket as well?
Please try and keep your changes as minimal as possible to only add the support for security context